Refactor SVN mission to fix 500 errors Issue #1398

willingc opened the pull request:
Most of this PR is additional comments and formatting changes (mostly for readability and some violate pep8 line length). The mission works properly locally.

phildini commented:
This looks solid, and the tests seem to agree! :D Thanks for adding all the comments as well.

:ship:, IMO.

willingc commented:
Squashed and rebased after phildini's review. Ready for @paulproteus to do the honors of merge and deploy. If it's still around in the morning, I'll do the honors :)

ehashman commented:
Woo very PEP8 :)
